server/src/repositories/usersRepository.js

```diff
@@ -9,4 +9,59 @@ export const usersRepository = {
 
     return user;
   },
+
+  signup: async (email, github_oauth_id) => {
+    const connection = database.connection.getRepository(Users);
+    const savedUser = await connection.save({
+      email,
+      github_oauth_id,
+      github_oauth_token: "",
+    });
+  },
+
+  updateGithubOauthToken: async (github_oauth_id, github_oauth_token) => {
+    const connection = database.connection.getRepository(Users);
+    const user = await connection.findOne({
+      where: { github_oauth_id, github_oauth_id },
+    });
+
+    user.github_oauth_token = github_oauth_token;
+    const savedUser = await connection.save(user);
+  },
+
+  getUser: async (id) => {
+    const connection = database.connection.getRepository(Users);
+    return await connection.findOne({ where: { id: id } });
+  },
+
+  getUserByGithubId: async (github_oauth_id) => {
+    const connection = database.connection.getRepository(Users);
+    return await connection.findOne({
+      where: { github_oauth_id: github_oauth_id },
```

Comment:
```suggestion
      where: { github_oauth_id },
```

이것도 이렇게 축약이 가능할겁니다 ㅎ